docs/releases/versions/version-v12.0.0.md

```diff
@@ -0,0 +1,117 @@
+- Record submission and inclusion in multiple communities
+- Display and dynamic administration of site banners
+- Creation of static pages via the administration panel
+- Instance-level configuration to enable metadata-only records and restricted files.
+- [New export formats](../../reference/export_formats.md) such as DACT-AP and GeoJSON
+- Contributed custom fields that are easy [to enable]((../../reference/metadata/optional_metadata.md))
+
+### Miscellaneous additions
+
+- Introduction of `PAGES_ALLOWED_EXTRA_HTML_ATTRS` and `PAGES_ALLOWED_EXTRA_HTML_TAGS` to allow specific HTML attributes and tags in Pages from [invenio-pages](PAGES_ALLOWED_EXTRA_HTML_TAGS)
+- Improvements to the `invenio rdm rebuild-all-indices` command
+- Mathematical formulas in a deposit can be rendered on the landing page via `THEME_MATHJAX_CDN = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"` and updated content-secuirty policy headers: add `cdnjs.cloudflare.com` to  `APP_DEFAULT_SECURE_HEADERS`.
+- Possibility to [add entry to a controlled vocabulary](../../customize/vocabularies/index.md#addupdate-fixtures-command) via the CLI: `invenio rdm-records add-to-fixture`
+
+## Breaking changes
+
+- See the [Requirements section](#requirements) below for what underlying requirements are now necessary
+- The [react-invenio-deposit module](https://github.com/inveniosoftware/react-invenio-deposit) has been moved to [invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records)
+    - Any imports should be replaced: `import { ... } from "react-invenio-deposit"` -> `import { ... } from "@js/invenio-rdm-records"`
+- Some `Overridable` React component ids may have changed. Verify that your overridden components load
+- `RoleNeed` uses the role's `id` instead of the role's `name`
+- The `/access/users` API URL has been renamed to `/access/grants` to manage access grants
+
+## Limitations and known issues
+TODO
+
+## Requirements
+
+InvenioRDM v12 now supports:
+
+- Python 3.9, 3.11 and 3.12
+- Node.js 18+
+- PostgreSQL 10+
+- OpenSearch v2
+
+Notably older versions of Elasticsearch/Opensearch and Node have been phased out.
+
+## Upgrading to v12.0
+
+We support upgrading from v11 to v12. See the [upgrade notice](../upgrading/upgrade-v12.0.md) for how.
+
+## Questions?
+
+If you have questions related to these release notes, don't hesitate to jump on [:simple-discord: discord](https://discord.gg/8qatqBC) and ask questions!
+
+## Credit
+
+The development work of this impressive release wouldn't have been possible without the help of these great people:
+
+- CERN: Alex, Anna, Antonio, Javier, Jenny, Karolina, Lars, Manuel, Nicola, Pablo G., Pablo P., Zacharias
+- Northwestern University: Guillaume
+- TU Graz: Christoph, David, Mojib
+- TU Wien: Max
+- Uni Bamberg: Christina
+- Uni Münster: Werner
+- Front Matter: Martin
+- KTH Royal Institute of Technology: Sam
```

Comment:
Caltech: Tom

Reply:
Thank you and apologies :bow: ! I just copied the list from another place and was also thinking: "there is going to be a ton more people here and I must be forgetting the usual suspects too" :smile: